modules/nw-ingress-set-or-delete-http-headers.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ingress-set-or-delete-http-headers_{context}"]
+= Setting or deleting HTTP request and response headers in an Ingress Controller
+
+Certain HTTP request and response headers might need to be set or deleted for compliance purposes or other reasons. You can set or delete these headers either for all routes served by an Ingress Controller or for specific routes.
+
+For example, you might want to migrate an application running on your cluster to use mutual TLS, which requires that your application check for an X-Forwarded-Client-Cert request header, but the {product-title} default Ingress Controller provides an X-SSL-Client-Der request header.
```

Comment:
"X-Forwarded-Client-Cert" and other headers should likely be in backticks, here and below

```suggestion
For example, you might want to migrate an application running on your cluster to use mutual TLS, which requires your application to check for an `X-Forwarded-Client-Cert` request header, but the {product-title} default Ingress Controller provides an `X-SSL-Client-Der` request header.
```